docs/api-reference/next/image.md

```diff
@@ -11,7 +11,9 @@ description: Enable Image Optimization with the built-in Image component.
   </ul>
 </details>
 
-> Before moving forward, we recommend you to read [Image Optimization](/docs/basic-features/image-optimization.md) first.
+Before moving forward, we recommend you to read [Image Optimization](/docs/basic-features/image-optimization.md) first.
+
+## Usage
 
 Image Optimization can be enabled via the `Image` component exported by `next/image`.
```

Comment:
Add a description to the page so a search shows it, also note that by having the recommended note as a note it'll be less prioritized for the description than the paragraph below it.

```suggestion
> Before moving forward, we recommend you to read [Image Optimization](/docs/basic-features/image-optimization.md) first.

Image Optimization can be enabled via the `Image` component exported by `next/image`.

## Usage

```

Reply:
Why do we suggest they read Image Optimization first? This should be in "Continued Reading" below.

Reply:
> Why do we suggest they read Image Optimization first

Because the other page explains what/where/how and this page is more of a reference page rather than an overview.